src/utilities/returnEnvironmentItems.js

```diff
@@ -28,12 +28,12 @@ export const returnJsonFileLocations = (fileName) => {
 
 	if (domain === "dev.baltimorecountymd.gov"){
 		fileLocation = _.filter(configItems.jsonFileLocationsTest, { key: fileName })
+	}else if(domain === "staging.baltimorecountymd.gov"){
```

Comment:
not blocking, but there is a lot of repeated code here, think we can improve in the future.